README.md

```diff
@@ -5,6 +5,8 @@ Welcome to the home page for the ICU4X-SC.  We are a subcommittee of ICU-TC in t
 
 Please subscribe to this repository to participate in discussions.  If you would like to join the core development team, apply to join [our mailing list](https://groups.google.com/a/unicode.org/forum/#!forum/omnicu-core).
 
+![Rust](https://github.com/echeran/icu4x/workflows/Rust/badge.svg?branch=master)
```

Comment:
Point to the upstream repo, not your fork

Reply:
Done, thanks.